gh-artifact-purge

```diff
@@ -14,6 +15,7 @@ FLAGS
   -d, --debug                          Enable debugging
   -h, --help                           Displays help usage
   -r, --run                            Apply changes; defaults to dryrun
+  -e, --expired                        Purge expired artifacts; defaults to skip
```

Comment:
Any concerns about renaming this?

```suggestion
  -d, --debug                          Enable debugging
  -h, --help                           Displays help usage
  -r, --run                            Apply changes; defaults to dryrun
  -p, --purge-expired                  Purge expired artifacts; defaults to skip
```

Aside from being alphabetical _(my OCD)_, I think having the short letter match the language we use is valuable for finding.